src/rard/templates/research/htmx_list_base.html

```diff
@@ -0,0 +1,44 @@
+{% load get_full_path_with_page %}
+
+<div id="list-container">
```

Comment:
```suggestion
<div id="pagination-list__container">
```

Reply:
Fair enough, but what's the double underscore mean?

Reply:
indicates the element level iirc - from [BEM](https://getbem.com/naming/)

Reply:
Ok done.